android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -245,7 +245,7 @@ public static void openBuySendSwapActivity(
         if (activityType == BuySendSwapActivity.ActivityType.SWAP_V2) {
             try {
                 BraveActivity.getBraveActivity().openNewOrSelectExistingTab(
-                        BraveActivity.BRAVE_SWAP_URL);
+                        BraveActivity.BRAVE_SWAP_URL, true);
```

Comment:
Why do we need to reload here? Is it because on re-opening the app we want to make sure we have the current lock state?

Reply:
It is for situation when you tried to open browser tab with address `brave://wallet/swap` and got empty page
but then you unlocked the wallet and pressed the `swap` button. 
without refresh, you will see the same empty tab `brave://wallet/swap`
But if you refresh it, all the info will be displayed